### PR TITLE
Add rig and maze to metadata

### PR DESCRIPTION
**Pull request by weiglszonja**
Extends metadata with rig and maze(s) table, based on the requirements [here](https://docs.google.com/spreadsheets/d/1BqmMxzZiaUyCvnOz2mF3XxCtkzu4xqDlPyQRVRqYLCU/edit#gid=0).

### CODE REVIEW

File: tests/test_ndx_tank_metadata.py [hunk 1]
```diff
@@ -12,6 +12,40 @@ def setUp(self):
...
+               'nidaqLines': str([0, 11]),
+               'syncClockChannel': '5',
+               'syncDataChannel': '6',
+               'rewardChannel': '0',
+               'rewardSize': '0.004',
+               'rewardDuration': '0.05',
+               'laserChannel': '1',
+               'rightPuffChannel': '2',
+               'leftPuffChannel': '3',
+               'webcam_name': 'Live! Cam Sync HD VF0770'}
```
Comment (bendichter) [inline tests/test_ndx_tank_metadata.py:40]: Several of these should be integers or floats, not strings

File: src/spec/create_extension_spec.py [hunk 2]
```diff
@@ -59,11 +60,49 @@ def main():
...
+    LabMetaDataExtension.add_group(
+        name='rig',
+        neurodata_type_inc='RigExtension',
+        doc='type for storing rig information',
+    )
+
+    MazeExtension = NWBGroupSpec(
+        doc='type for storing maze information',
+        neurodata_type_def='MazeExtension',
+        neurodata_type_inc='DynamicTable',
```
Comment (bendichter) [inline src/spec/create_extension_spec.py:94]: Why a dynamic table?

Comment (weiglszonja) [reply src/spec/create_extension_spec.py:94]: Because it can have several rows, depends on each file how many.

**Comment by bendichter** [comment]
Need to create object mapper. See https://github.com/NeurodataWithoutBorders/pynwb/blob/9d32f5efad9f3b977eb5bff1ef84efca785d5d8a/src/pynwb/io/file.py#L143-L147

**Comment by bendichter** [comment]
Need code for reading from .mat file and writing data to NWBFile object using this extension

**Comment by bendichter** [comment]
https://stackoverflow.com/questions/15512560/access-mat-file-containing-matlab-classes-in-python

**Pull request merged**